protocols/migration/migration_helpers.py

```diff
@@ -100,9 +109,9 @@ def reverse_migrate_interpretation_request_rd_to_v3(json_dict, old_ig, cip=None)
             lambda x: x,
             MigrateReports400To300().migrate_interpretation_request_rd,
             lambda x: MigrateReports500To400().migrate_interpretation_request_rd(x, old_ig=old_ig, cip=cip),
```

Comment:
`old_ig` is not defined any more

Reply:
Sorry that was part of my ongoing merge hell